pages/dkp/kommander/2.2/install/air-gapped/index.md

```diff
@@ -213,16 +213,10 @@ Based on the network latency between the environment of script execution and the
       helmMirrorImageTag: "${VERSION}-amd64"
     ```
 
-1. Download the Kommander application definitions:
-
-    ```bash
-    wget "https://downloads.mesosphere.com/dkp/kommander-applications_${VERSION}.tar.gz"
-    ```
-
 1. To install Kommander in your air-gapped environment using the above configuration file, enter the following command:
 
     ```bash
-    kommander install --installer-config ./install.yaml --kommander-applications-repository kommander-applications_${VERSION}.tar.gz
+    kommander install --installer-config ./install.yaml
```

Comment:
This deletion is confusing to me...  We published a kommander-applications_v2.1.1.tar.gz when 2.1.1 shipped.   Where is it used if not here?   Or is this planned to change again in 2.2???

Also, this --kommander-applications-repository flag mention is missing from the 2.1 docs as well...  

?????

Reply:
I think this PR was just caught mid-merge or something like that.
I'll re-add

Reply:
updated